src/modules/p_usrloc/ul_db_watch.c

```diff
@@ -96,7 +96,10 @@ void check_dbs(unsigned int ticks, void *param){
 	ul_db_handle_list_t * tmp2, * new_element;
 	int found;
 	int i;
-	
+
+	if(mdb_availability_control) {
```

Comment:
mdb_availability_control is also checked inside check_master_db() function.

Reply:
correct, i m removing the one inside the check_master_db() function.